zeppelin-web/src/app/notebook/notebook.css

```diff
@@ -37,6 +37,34 @@
   margin-top: 20px;
 }
 
+/*
+  Add New Paragraph
+ */
+
+.new-paragraph{
+  text-align:center; 
+  height:7px;
+  margin:-7px 2px 0 2px;
+  cursor: pointer;
+}
+
+.new-paragraph:hover{  
+  background-color: #d3e5ea;
```

Comment:
Could you change it to ``background: #d3e5ea``?
There is already a PR taking fixing those, so we don't want to add more